app/controllers/wishlists_controller.rb

```diff
@@ -37,4 +54,10 @@ def wishlist_params
     params[:wishlist][:user_id] = current_user.id
     params.require(:wishlist).permit(:user_id, :name, :description)
   end
+
+  def find_wishlist
+    @wishlist = Wishlist.find(params[:id])
+  rescue ActiveRecord::RecordNotFound
+    redirect_to wishlists_path, alert: t("generic_error")
```

Comment:
and in this case method name could be even `set_wishlist!` to emphasize that exception will be raised